src/main/webapp/style.css

```diff
@@ -13,4 +13,7 @@
 #video-context {
   height: 300px;
   width: 300px;
+}
+#nav-tabContent {
+  margin-top:8px;
```

Comment:
This is a design changes long time ago from the screenshot Cis showed us:)

Reply:
No worries! https://screenshot.googleplex.com/Xqy9wJjHVUd

Reply:
Not a big change

Reply:
hahah it was kinda a joke I think... But if it looks good then sure!